3.0.0/README.md

```diff
@@ -0,0 +1,327 @@
+
+```JSON
+{
+  "data": [
+    "http://www.example.com/admin/data.geojson"
+  ]
+}
+```
+
+## 3.8 `description`
+
+OPTIONAL. Default: null.
+
+A text description of the set of tiles. The description can contain any valid unicode character as described by the JSON specification [RFC 8259](https://tools.ietf.org/html/rfc8259).
+
+```JSON
+{
+  "description": "We are volcanoes. When we women offer our experience as our truth, as human truth, all the maps change. There are new mountains."
+}
+```
+
+## 3.9 `fillzoom`
+
+OPTIONAL. Default: null.
+
+An integer specifying the zoom level at which to generate overzoomed tiles from. Implementations may generate overzoomed tiles from parent tiles if the requested zoom level does not exist. In most cases, overzoomed tiles are generated from the maximum zoom level of the set of tiles. If fillzoom is specified, the overzoomed tile is generated from the fillzoom level.
+
+For example, in a set of tiles with maxzoom 10 and _no_ fillzoom specified, if a request for a z11 tile comes through, the implementation will use the maximum z10 parent tiles to generate the new, overzoomed z11 tile. If the same TileJSON object had fillzoom specified at z7, a request for a z11 tile would use the z7 tile instead of z10.
+
+```JSON
+{
+  "fillzoom": 7
+}
+```
+
+## 3.10 `grids`
+
+OPTIONAL. Default: [].
+
+An array of interactivity endpoints. {z}, {x} and {y}, if present, are replaced with the corresponding integers. If multiple endpoints are specified, clients may use any combination of endpoints. All endpoints MUST return the same content for the same URL. If the array doesn't contain any entries, UTF-Grid interactivity is not supported for this set of tiles. See https://github.com/mapbox/utfgrid-spec/tree/master/1.2 for the interactivity specification.
+
+*Note: UTF-Grid interactivity predates GL-based map rendering and interaction. Map interactivity is now generally defined outside of the TileJSON specification and is dependent on the tile rendering library's features.*
+
+```JSON
+{
+  "grids": [
+    "http://www.example.com/earthsea/1.0.0/{z}/{x}/{y}.grid.json"
+  ]
+}
+```
+
+## 3.11 `legend`
+
+OPTIONAL. Default: null.
+
+Contains a legend to be displayed with the map. Implementations MAY decide to treat this as HTML or literal text. For security reasons, make absolutely sure that this field can't be abused as a vector for XSS or beacon tracking.
+
+```JSON
+{
+  "legend": "Dangerous zones are red, safe zones are green"
+}
+```
+
+## 3.12 `maxzoom`
+
+OPTIONAL. Default: 30. >= 0, <= 30.
+
+An integer specifying the maximum zoom level. MUST be >= minzoom.
+
+```JSON
+{
+  "maxzoom": 11
+}
+```
+
+## 3.13 `minzoom`
+
+OPTIONAL. Default: 0. >= 0, <= 30.
+
+An integer specifying the minimum zoom level. MUST be <= maxzoom
```

Comment:
as a counterpart to mentioning overzooming when defining maxzoom, maybe touch on the impossibility of underzooming here?

Reply:
Underzooming isn't impossible, afaik. It's just that mapbox-gl-js doesn't support it.

Reply:
True, theoretically underzooming could be possible. 

My reasoning for making this suggestion (and the suggestion above about overzooming) is that I know users of tileJSON are often confused by overzooming. Tiles _beyond_ the maxzoom can be requested, while tiles _below_ the minzoom cannot. Maybe the _correct_ solution is to incorporate a description of how far tiles overzoom/underzoom into tileJSON – this sounds like something we should save until v4.

Reply:
> Tiles beyond the maxzoom can be requested

I think this is outside of the scope of the specification itself to enforce, and is a choice made by a server, which can choose to use maxzoom or fillzoom as the z level to overzoom from. I think there's room to mention overzooming _somewhere_ in the spec, though, as an aid to those reading. Maybe worth creating an FAQ section to refer to?